nrf_security/README.rst

```diff
@@ -11,12 +11,16 @@ This module includes an Mbed TLS glue layer to enable both hardware-accelerated
    The nrf_security module interfaces with the :ref:`nrf_cc3xx_mbedcrypto_readme`.
    This library conforms to a specific version of Mbed TLS.
 
+Prerequisites
+*************
+
+Mbed TLS is required and usually provided by west, but it can also be
+manually configured with the CMake variable ``ARM_MBEDTLS_PATH``.
```

Comment:
Should we specify that the `ARM_MBEDTLS_PATH` should point to the top-level folder of the arm Mbed TLS repository ?
Or is that obvious.

@frkv thoughts ?

Reply:
That is obvious IMHO.

Reply:
This should be obvious, agree with @SebastianBoe